scraper/domainScraper/pipelines/mongo.py

```diff
@@ -81,15 +82,28 @@ def buildAI_SentimentPayload(sentimentArray, target):
             for sentiment in sentimentArray:
                 for key, value in sentiment.items():
                     if key in self.payload[target]:
-                        print(self.payload[target][key])
                         self.payload[target][key]['accuracy'] = (self.payload[target][key]['accuracy'] + sentiment[key]['accuracy']) / 2
                         self.payload[target][key]['Total'] = self.payload[target][key]['Total'] + sentiment[key]['Total']
-                        print(self.payload[target][key])
                     else:
                         self.payload[target][key] = value   
                     
            # print("Payload: ", self.payload[target])
 
         buildAI_SentimentPayload(item['AI_Sentiment'], 'AI_Sentiment')
+        
+        def buildWebClassificationPayload(classification, target):
 
+            for key, value in classification.items():
+                if key in self.payload[target]:
+                    # Calculate accuracy average
+                    self.payload[target][key]['accuracy'] = (self.payload[target][key]['accuracy'] + classification[key]['accuracy']) / 2
+                    # Calculate total hits 
+                    self.payload[target][key]['Total'] = self.payload[target][key]['Total'] + classification[key]['Total']
```

Comment:
Please consider using `buildPayload` function for calculating totals, I think it might not be clear because of the variable/function naming, but it can be used in any collection where you're summing totals, think of it as an accumulator I suppose. And since you also want to find the accuracy for sentiment analysis and classification, you could create a function that you could call for both.

Also something to think about, what does your function do? It sums the total and it calculates the average accuracy of the data. Because there is an and in that sentence, you should split the functionality into separate functions at each and. So you want a function that sums totals of data, in this case you can use `buildPayload`  and calculate the average accuracy of the data.

Reply:
ok, seems reasonable I'll make the changes